build.gradle

```diff
@@ -489,7 +489,7 @@ tasks.cftlibTest.dependsOn(installChromium)
 generateCCDConfig.doLast {
   copy {
     from file('ccd-definitions')
-    into layout.buildDirectory.dir('definitions/NFD')
+    into layout.buildDirectory.dir('definitions/NFD-4140')
```

Comment:
Changes to the case type number are for testing in preview. I'll remove these before merging